### ISSUE TITLE
`svgbobdoc` is not compatible with derive macro which uses `syn 2.0`

### ISSUE THREAD

**Issue by CurryPseudo** [yvt/svgbobdoc#7]
For example `serde >=1.0.157` or `thiserror >= 1.0.40`
See https://github.com/serde-rs/serde/issues/2413.

**Comment by yvt** [comment]
The same error occurs with other macros:

```rust
#[doc = concat!("a", "b")]
#[derive(serde::Serialize)]
pub struct Foo {}
```

```
error: unexpected token
  --> example/src/lib.rs:89:15
   |
89 | #[doc = concat!("a", "b")]
   |               ^
```

Seems like a `syn` issue.

**Comment by yvt** [comment]
Opened https://github.com/dtolnay/syn/issues/1418

A temporary work-around is to add `syn ^2` to `build-dependencies` with `full` feature enabled.

**Comment by yvt** [comment]
Fixed in `syn 2.0.5`
